src/crud.js

```diff
@@ -0,0 +1,84 @@
+import { loadTodos, updateIndex, uploadTodo } from './helpers.js';
+
+const formEl = document.querySelector('.input-container');
+const todoInputEl = document.querySelector('.todo-input');
+const listsEl = document.querySelector('.lists');
+
+const renderTodo = () => {
+  const tasks = loadTodos();
+  if (!tasks) return;
+
+  const tasksSorted = tasks.slice().sort((a, b) => a.index - b.index);
+  listsEl.innerHTML = '';
+  tasksSorted.forEach((task) => {
+    const { completed } = task;
+    const flag = completed ? 'checked' : '';
+    const cross = completed ? 'cross' : '';
+    const markup = `
+      <li class="list">
+      <span class="list-left">
+        <input type="checkbox" class="check" data-check="${task.index}" ${flag}/>
+        <input type="text" data-desc="${task.index}" class="todo todo-${task.index} ${cross}" value="${task.description}"/>
+      </span>
+      <button class="btn remove" data-remove="${task.index}">
+      <img
+      src="https://e7.pngegg.com/pngimages/179/938/png-clipart-computer-icons-hamburger-button-dots-kebab-menu-text-rectangle-thumbnail.png"
+    />
```

Comment:
## Suggestion
- [ Optional Change ] It would be nice if you use a trash icon for the delete functionality you placed on this three-dotted icon. The three-dotted icon is meant for options but the functionality you place there is for deleting a todo and the best icon should be a delete or trash icon. KIndly work on that 👍 

![image](https://user-images.githubusercontent.com/50941074/179323133-b744205f-5ad3-4a7e-bfc0-d8d0c62f4ba9.png)